src/pymovements/datasets/colagaze.yaml

```diff
@@ -0,0 +1,63 @@
+name: "CoLAGaze"
+
+has_files:
+  # need to fix bug:
+  #     ValueError: Experiment metadata does not match the metadata in the ASC file:
+  #     - Right eye tracked: True != False
+  gaze: false
+  precomputed_events: true
+  precomputed_reading_measures: true
+
+mirrors:
+  gaze:
+      - https://files.au-1.osf.io/v1/resources/gj2uk/providers/osfstorage/67e14ce0f392601163f33215/?view_only=a8ac6e0091e64d0a81d5b1fdec9bab6e&zip=
```

Comment:
so this means the dataset needs to be downloaded three times? how big are the archives?

Just another sign that the current design of `DatasetDefinition` is really bad and needs to be improved.

Reply:
ah, so let's fix it in logic. can you create an issue?

Reply:
#1136